tags/tags.go

```diff
@@ -75,5 +75,24 @@ func Highway() map[string]bool {
 	tags["service"] = false
 	tags["tertiary"] = false
 	tags["road"] = false
+	tags["pedestrian"] = false
+	tags["unclassified"] = false
+	tags["living_street"] = false
+	tags["track"] = false
```

Comment:
could we please leave the default tags the way they were before, to avoid a breaking change?